client/components/forms/core/components/VSelect.vue

```diff
@@ -4,230 +4,239 @@
     class="v-select relative"
     :class="[{ 'w-0': multiple, 'min-w-full': multiple }]"
   >
-    <div
-      class="w-full flex overflow-hidden"
-      :style="inputStyle"
-      :class="[
-        theme.SelectInput.input,
-        theme.SelectInput.borderRadius,
-        { 
-          'ring-red-500! ring-2! border-transparent!': hasError, 
-          '!cursor-not-allowed bg-neutral-200! dark:bg-neutral-800!': disabled,
-          'focus-within:ring-2 focus-within:ring-form/100 focus-within:border-transparent': !hasError && !disabled
-        },
-        inputClass
-      ]"
+    <UPopover
+      v-model:open="isOpen"
+      :content="{
+        align: 'start',
+        side: 'bottom',
+        sideOffset: 4
+      }"
+      :ui="{
+        content: 'w-(--reka-popper-anchor-width) bg-white dark:!bg-notion-dark-light shadow-xl z-30 overflow-auto ' + [theme.SelectInput.dropdown, theme.SelectInput.borderRadius].join(' ')
```

Comment:
Where did you find `--reka-popper-anchor-width`? I could only find `--reka-popover-trigger-width` on https://reka-ui.com/docs/components/popover

Reply:
https://ui.nuxt.com/components/popover#with-anchor-slot

<img width="1822" height="1034" alt="CleanShot 2025-09-03 at 11 09 44@2x" src="https://github.com/user-attachments/assets/691d0c4a-6d9b-42c4-b4f9-6a432814b41a" />